content/tutorials/esc-version-tagging/index.md

```diff
@@ -0,0 +1,254 @@
+---
+title: Configure applications dynamically with ESC version tagging
+title_tag: Configure applications dynamically with ESC version tagging
+layout: single
+description: Use immutable versioning and dynamic tagging to configure multiple applications safely and easily with Pulumi ESC.
+meta_desc: Use immutable versioning and dynamic tagging to configure multiple applications safely and easily with Pulumi ESC.
+meta_image: meta.png
+weight: 999
+summary: |
+    As a hosted service of Pulumi Cloud, [Pulumi ESC](https://pulumi.com/esc) makes it easy to store and retrieve static and dynamic configuration settings --- including encrypted secrets --- manage them securely and flexibly, and use them in your applications in various ways. In this tutorial, you'll use ESC's [version tagging](/docs/esc/environments/versioning/) features to configure multiple running Node.js applications with a single shared configuration value referenced by tag name.
+youll_learn:
+    - How to store and retrieve configuration values with Pulumi ESC
+    - How to use environment versions to pin applications to specific config values
+    - How to use version tagging to make configuring applications safer and more flexible
+    - How to embed and use Pulumi ESC in your applications with the ESC SDK for Node.js
+prereqs:
+    - Either the standalone [Pulumi ESC CLI](/docs/esc/install/) or the [Pulumi CLI](/docs/iac/install/)
+    - A [Pulumi Cloud account](https://app.pulumi.com/signup) and [access token](/docs/pulumi-cloud/accounts/#access-tokens)
+    - A Pulumi Cloud organization with a subscription to the [Enterprise edition](https://pulumi.com/pricing) (or above)
```

Comment:
I feel like we need a way to distinguish tutorials/docs about paid features at the title-level. e.g. maybe have a front-matter that flags it and shows a special icon or something.

Reply:
Agree, yep. Filed: https://github.com/pulumi/docs/issues/12931